include/nbl/ui/CWindowManagerWin32.h

```diff
@@ -20,7 +20,26 @@ namespace nbl::ui
 	public:
 		CWindowManagerWin32() = default;
 		~CWindowManagerWin32() {};
-
+		virtual core::smart_refctd_ptr<IWindow> createWindow(IWindow::SCreationParams&& creationParams) override
+		{
+			CWindowWin32::native_handle_t handle = createNativeWindow(creationParams.x,
+				creationParams.y,
+				creationParams.width,
+				creationParams.height,
+				creationParams.flags,
+				creationParams.windowCaption);
+			// TODO: params validation
```

Comment:
also return nullptr in case of failed native handle creation

Reply:
@sadiuk TODO

Reply:
still a tODO?